lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1766,6 +1766,11 @@ def _parse_marketplace_image(
                     offer=marketplace.offer,
                     skus=marketplace.sku,
                 )
+            if 0 == len(versioned_images):
+                raise LisaException(
+                    f"fail to list versions of image {marketplace.publisher} "
```

Comment:
Make the "fail" accurate, like "cannot find any version of ..."

Reply:
fixed